client/apps/edit2/components/admin/components/metaphysics_autocomplete.jsx

```diff
@@ -0,0 +1,130 @@
+import request from 'superagent'
+import { clone, uniq } from 'lodash'
+import { connect } from 'react-redux'
+import { difference, flatten, pluck } from 'underscore'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { onChangeArticle } from 'client/actions/editActions'
+import { AutocompleteList } from '/client/components/autocomplete2/list'
+import * as Queries from 'client/queries/metaphysics'
+
+export class MetaphysicsAutocomplete extends Component {
```

Comment:
Don't _love_ the naming here -- maybe `AutocompleteListMetaphysics`? Something to indicate a list would be nice.

Reply:
👍